uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/document-list.spec.js

```diff
@@ -143,8 +145,136 @@ define([
         });
 
         it('formats and creates date range label containing until date only', function () {
-          expect($scope.label.dateRange).toBe('Until: ' +  $filter('date')($scope.filterParams.dateRange.until, 'dd/MM/yyyy'));
+          expect($scope.label.dateRange).toBe('Until: ' + $filter('date')($scope.filterParams.dateRange.until, 'dd/MM/yyyy'));
+        });
+      });
+    });
+
+    describe('sortBy', function () {
+      beforeEach(function () {
+        initController();
+
+        _.each(documentFabricator.documentStatus(), function (option) {
+          $rootScope.cache.documentStatus.obj[option.key] = option.value;
+        });
+
+        _.each(documentFabricator.documentTypes(), function (option) {
+          $rootScope.cache.documentType.obj[option.key] = option.value;
+        });
+
+        _.each(contactFabricator.list(), function (option) {
+          $rootScope.cache.contact.obj[option.contact_id] = option;
+        });
+
+        $rootScope.cache.assignmentType.obj = assignmentFabricator.assignmentTypes();
+        $rootScope.cache.assignment.obj = assignmentFabricator.listAssignments();
+      });
+
+      describe('documents are not sorted', function () {
+        it('lists unsorted document list', function () {
+          expect($scope.list[0].id).toBe('1200');
```

Comment:
This test is not clear to me, why are we testing with two arbitrary number 1200 and 1213.

Reply:
Re-written tests.